dcp/DcpHandler.go

```diff
@@ -322,8 +322,8 @@ func (dh *DcpHandler) checkColMigrationDataCloned(mut *Mutation) {
 	dummyReq.Req = &gomemcached.MCRequest{}
 	matchedNamespaces, errMap, errMCReqMap := dh.migrationMapping.GetTargetUsingMigrationFilter(uprEvent, dummyReq, dh.logger)
 	if len(matchedNamespaces) > 1 {
-		dh.logger.Debugf("Document %s with opCode %v matched more than once: %v, errMap %v, errMCReqMap %v",
-			uprEvent.UprEvent.Key, uprEvent.UprEvent.Opcode, matchedNamespaces.String(), errMap, errMCReqMap)
+		dh.logger.Debugf("Document %s (%v) with length %v opCode %v matched more than once: %v, errMap %v, errMCReqMap %v",
+			uprEvent.UprEvent.Key, uprEvent.UprEvent.Key, len(uprEvent.UprEvent.Key), uprEvent.UprEvent.Opcode, matchedNamespaces.String(), errMap, errMCReqMap)
```

Comment:
It may be more human-readable to print the key both as a string and as hex.
Printing a bunch of binary numbers aren't that useful.